src/selectors/entities/posts.js

```diff
@@ -483,3 +467,37 @@ export function getPostIdsInChannel(state, channelId) {
 
 export const isPostIdSending = (state, postId) =>
     state.entities.posts.sendingPostIds.some((sendingPostId) => sendingPostId === postId);
+
+export const makeIsPostCommentMention = () => {
+    return createSelector(
+        getAllPosts,
+        getPostsInThread,
+        getCurrentUser,
+        (state, postId) => getPost(state, postId),
```

Comment:
You can have this just as getPost without the arrow function

Reply:
Yup that's true. Changed it to have just the selector.